pkg/logger/zap.go

```diff
@@ -15,12 +15,15 @@ import (
 // The package logger can only be init once, so subsequent calls to this method
 // won't have any effect.
 func InitZap(level int, opts ...LoggerOpt) error {
-	cfg := zap.NewDevelopmentConfig()
-	cfg.Level = zap.NewAtomicLevelAt(zapcore.Level(-1 * level))
+	filename := fmt.Sprintf("./eksa-cli-%s.log", time.Now().Format("2006-01-02T15:04:05"))
+	cfg := Config{
+		Level:         NewAtomicLevelAt(level),
+		EncoderConfig: zap.NewDevelopmentEncoderConfig(),
+		OutputPaths:   []string{filename},
```

Comment:
IMO this should be optional, not by default

Reply:
oh I'm not saying the CLI should be able to configure it (at least for now)
But that it should be optional when innit the zap logger

Reply:
Oh yeah, the logger is initialized in a few different places and they all don't need this. I've updated adding an output path to be optional.